packages/fxa-support-panel/src/remote-lookup/remote-responses.dto.ts

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Note that these `*.Response` interfaces are purely for access to known
+// response keys and not an attempt to validate the return payloads from
+// fxa-auth-db-mysql.
+export interface AccountResponse {
```

Comment:
Is there much value in having these interfaces in fxa-shared? Would we be able to use this in fxa-settings?

Reply:
It is possible to generate a type for the response from a `joi` schema using `typesafe-joi`, so we could move the validation schemas to fxa-shared, and then have usable types everywhere. However, we're also moving direct db access into fxa-shared, so when that's in there, the support panel could use that directly instead of these calls into auth-db-mysql.